### DISCUSSION TITLE
All those, who started and supported anti RMS movement - should be retroactively canceled from free software world for extremism and false accusations!

### DISCUSSION THREAD

**Discussion by AnimusPEXUS** [rms-support-letter/rms-support-letter.github.io#1821] [General]
thumbs up!

**Comment by shiftee** [comment]
Thumbs down. 
Hopefully RMS remains on the board we can all go back to making software.
Otherwise we may need to fragment like many religions and form the Reformed Free Software Foundation :)

**Reply by AnimusPEXUS** [reply]
imo, such way of thought is kind of cancer ignoring - you ignore it and then it kills you

**Reply by shiftee** [reply]
I can see where you're coming from but sometimes treating a cancer will kill you anyway.
Occasional social media drama is something we unfortunately need to deal with nowadays, it needs to be de-escalated or even better, ignored.

**Reply by pda0** [reply]
You cannot put out a fire with gasoline. Unity cannot be achieved through separation. You can't stop the culture of cancellation by practicing cancellation.
Only by resisting animal instincts will we get a chance to stop the wave of thoughtless cruel online-reprisals. Those who put signatures against Stallman are not enemies. Many of them are people who repeat other people's words without bothering to check them and reflect on what they have read. But one day they may think and regret their actions.

**Reply by AnimusPEXUS** [reply]
> Unity cannot be achieved

I don't feel like it is ok to unite with people, who specially searching reasons to harass other people.. - this is in-productive. and I personally think, separation is unavoidable - this is better, than bear with people who constantly wishes bad for you.

**Comment by ahangarha** [comment]
Rather trying to remove people, I would advocate the approach to be more proactive in supporting the cause in what ever capacity one person can do. Either as a programmer or merely being a blogger, one should take the activity seriously. It is a moral duty on out shoulder to keep the flag up it the responsibly gets increased after such incidents take place.

My strong suggestion is to focus on work rather than cutting sick minded people (that no one can even do practically)

**Comment by thatHguy** [comment]
I don't know if they should be cancelled - that's just using their own vile mob games against them. But they should definitely and without doubt be completely ashamed of themselves! 

Also keep in mind that the bulk of the "mob" are moronic journalists and Twitter idiots, with just enough prominent names pushed in there by organizations with an agenda to get noticed. Their memory spans extend as far as the previous 48 hours of their lives, and "free software" to them just means Candy Crush for their phone. In other words most of them haven't got a clue.

Some on the anti-FSF camp are open-source projects backed by huge companies and Big Tech. There is vested motivation and incentive here. The libre movement has been too naive. The classic "embrace, extend, extinguish" is being played out, with this being the beginning of the "extinguish" phase.

**Comment by waltercool** [comment]
I don't agree on cancel culture for sure, so I can't agree on this. That would make tensions even worse.

Now, take notes of every company trying to cancel "RMS with the FSF board members" and make them disappear from your existence. That means, don't recommend them on professional or personal advice and donate to other organizations.

While I'm okay with individuals doing this, seeing Linux distros like Debian, foundations like Gnome/EFF/Mozilla/Tor/FPF/OSC and specially companies like Redhat/Suse supporting the coup is not okay.

And it's all sad, I donated in past to many of those foundations and distros, I supported many times those companies! And watching them being so awfully evil is completely disgusting, the Brownshirt/Bolsheviks game must not be tolerated.

So again, don't be against them, don't play their same stupid authoritarian game. Just ignore them, make like they don't exist. If you see some friend or co-worker using their tools, suggest them others.

**Comment by artemgur** [comment]
So-called "cancel culture" is an unacceptable abomination which cannot be tolerated. *No one* should be harassed or ostracized for their words *ever*. That's the point of freedom of speech in the first place. Freedom of speech which is supposed to be the basis of the society and one of the most basic human rights. And *where* is the freedom of speech *now*?
Harassment like that heavily reminds me the harassment in autoritarian countries. Like, you know, when Pasternak (Soviet writer) was harassed into refusing the Nobel prize. When you are harassed in every media (newspapers mostly then).
But the harassment like of Stallman feels *way* more dirty to me. When you are harassed by opportunists who want either to create scandal to get popularity and fame in media or to actively undermine the very cause of free software, and by mob. And, what is the most abominable and scary, ones who are *afraid*. Afraid to lose public image or be harassed.

Tldr: the "cancel culture" is the only thing that needs to be cancelled. And as soon as possible.

**Reply by Bluesboy** [reply]
If Pasternak weren't harassed he could became traitor like Solzhenitsyn. So he's a bad example. Nobel Prize it a tool for capitalists.
Whole SJW movement,"cancel culture", diversity and PC are also tools but of other kind. They are also used by capitalists and corporations.
